d2core/d2input/input_manager.go

```diff
@@ -114,8 +124,8 @@ func (im *inputManager) advance(elapsed float64) error {
 		})
 	}
 
-	for button := ebiten.MouseButtonLeft; button < ebiten.MouseButtonMiddle; button++ {
-		if inpututil.IsMouseButtonJustPressed(button) {
+	for button := mouseButtonMin; button < mouseButtonMax; button++ {
```

Comment:
I don't know about `mouseButtonMin` and `mouseButtonMax`.  Its not really clear what they refer to.  We might support 5+ button mice, so assigning to the right button doesn't seem right.

Reply:
I'm OK with this, we haven't really defined key binding support yet.